### ISSUE TITLE
[LocalAuthentication][9.2.0] Must have com.samsung.android.bio.face.permission.USE_FACE

### ISSUE THREAD

**Issue by Bardiamist** [expo/expo#9927]
## 🐛 Bug Report

Users who preferred `Face recognition` in system `Settings` -> `Biometrics and security` ->`Biometric preferences` -> `Preferred biometric` can't use `LocalAuthentication.authenticateAsync`.

### Environment

  Expo CLI 3.24.2 environment info:
    System:
      OS: macOS 10.15.6
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 14.8.0 - /usr/local/bin/node
      Yarn: 1.22.4 - /usr/local/bin/yarn
      npm: 6.14.7 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 4.0 AI-193.6911.18.40.6626763
      Xcode: 11.6/11E708 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.13.1 => 16.13.1 
      react-native: 0.63.2 => 0.63.2 
    npmGlobalPackages:
      expo-cli: 3.24.2

Android

"expo-local-authentication": "9.2.0"

### Steps to Reproduce

On Samsung Galaxy S8 with Android 9:
1. Enable `Face recognition` in `Settings` -> `Biometrics and security` ->`Biometric preferences` -> `Preferred biometric`
2. Try to call `LocalAuthentication.authenticateAsync`

### Expected Behavior

Fallback to fingerprint

### Actual Behavior

Crash even use try/catch with `Must have com.samsung.android.bio.face.permission.USE_FACE or android.permission.USE_BIOMETRIC permission.` error

### Reproducible Demo

https://snack.expo.io/@bardiamist/localauthentication

### Stack trace

https://sentry.io/share/issue/0a5f845d1e27427f95512515b0b1a86c/

### Additional info
Previous issue #8530 was closed. Seems that PR #8692 wasn't tested and definitely don't fixed issue. Tested on `expo-local-authentication@9.2.0` issue still exist.

**Comment by byCedric** [comment]
Hi @Bardiamist. We do test our releases and updates, so I'm not sure why you feel that PR #8692 wasn't tested. Luckily, this is just a missing permission error and easily fixable if you include `USE_BIOMETRIC` in the [app manifest's `android.permissions` list](https://docs.expo.io/versions/v38.0.0/config/app/#permissions). I'm assuming you declared this because without that the permission is included by default (because PR #8692 fixed that).

It's normal for both Android and iOS apps to crash when they don't have the required permissions added. Although I agree with you that this is extreme and actually shouldn't happen, it's something we can't fix because it's OS related.

Hope this helps!

**Comment by Bardiamist** [comment]
@byCedric 

>so I'm not sure why you feel that PR #8692 wasn't tested

Because you not answered in #8692 and not confirmed that it was tested.

I have bare project and also have
```
<uses-permission android:name="android.permission.USE_BIOMETRIC" android:requiredFeature="false" />
```
in `AndroidManifest.xml`.

Did you test that on Samsung with enabled Face recognition (in Settings -> Biometrics and security -> Biometric preferences -> Preferred biometric)? If yes then what device did you use? Seems that case not tested and not fixed. Slill crash on Samsung Galaxy S8 with Android 9.
